arch/x86/core/cache.c

```diff
@@ -18,32 +18,40 @@
 #include <zephyr/cache.h>
 #include <stdbool.h>
 
+/* Not Write-through bit */
```

Comment:
I'll push a white list on the relevant PR. remove that commit.

Reply:
The commit would be rebased when you PR is merged. Without this I cannot compile and test the code.